progression/src/main/java/me/mykindos/betterpvp/progression/profession/skill/woodcutting/TreeCompactor.java

```diff
@@ -100,6 +107,24 @@ public void onPlaceCompactedLog(BlockPlaceEvent event) {
         }
     }
 
+    /**
+     * cancel crafting compacted logs into wooden planks
+     */
+
+    @EventHandler
+    public void onCompactedLogCraft(CraftItemEvent event) {
+        ItemStack result = event.getRecipe().getResult();
+        if (!(result.getAmount() == 4 && result.getType() == Material.OAK_PLANKS)) return;
```

Comment:
just do != 4 and != oak planks for better readability

Reply:
done